src/commands/diagnostics/logging/util.ts

```diff
@@ -0,0 +1,33 @@
+import { DiagnosticLoggingStatus, LoggingLevel } from '../../../sdk';
+import { OutputFormatter } from '../../../util/formatter';
+
+export const getLevels = (enumObject: typeof LoggingLevel) : string[] => {
+	let levels = [];
+	for (let key in enumObject) {
+		if ({}.hasOwnProperty.call(enumObject, key)) {
```

Comment:
This is weird but it gets rid of an eslint warning

Reply:
Try using this:
```
export const getLogLevels = () : string[] => {
  return Object.keys(LoggingLevel).map(key => LoggingLevel[key]); // possibly use Object.values(LoggingLevel)?
}
```
https://stackoverflow.com/questions/39372804/how-can-i-loop-through-enum-values-for-display-in-radio-buttons

Reply:
Might be worth moving this to models as well so the UI can use this function and not have duplicate logic